force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -126,16 +144,17 @@ public with sharing class ClassesObjectsMethods {
         List<String> scientistNames = new List<String>{'Tim Berners-Lee', 'Alan Turing', 'Grace Hopper', 'Donald Knuth', 'Guido van Rossum', 'Ken Thompson', 'Stephen Hawking'};
 
         // The variable to store the concatenated string
-
+        String concatenatedOutput = '';
         // Loop through the list of scientists' names
-
             // If the name doesn't contain the letter 'G', skip this iteration
-
             // Add the name to the result string, followed by a comma
-
         // Remove the last comma from the result string
-
-        return null; // Replace null with the variable you used to store the result
+        for(String currentName : scientistNames) {
+            if(currentName.contains('g') || currentName.contains('G')) {
```

Comment:
Here we can shorten things up by using the `containsIgnoreCase()` method:
```suggestion
            if (currentName.containsIgnoreCase('g')) {
```
Or we could use toLowerCase():
`if (currentName.toLowerCase().contains('g')){`